__tests__/add-remove.test.js

```diff
@@ -0,0 +1,99 @@
+import { addNewTodo, onDeleteTodo } from '../src/add-remove.js';
+import { localStorageMock } from '../__mocks__/mockStorage.js';
+
+describe('Todo', () => {
+  describe('Add Todo', () => {
+    test('task 1 should be equal {description; task 1, completed: false, index: 1} ', () => {
+      const task = 'task 1';
+      const todos = [];
+      const expectedTodo = { description: 'task 1', completed: false, index: 1 };
+      const newTodo = addNewTodo(task, todos);
+      expect(newTodo).toEqual(expectedTodo);
+    });
+  });
+
+  describe('Delete Todo', () => {
+    test('task 1 should be equal {description; task 1, completed: false, index: 1} ', () => {
```

Comment:
- [ ] I think this `delete todo` test description does not suits well. It supposed to be like `todo will reduced to 0 if we have one task earlier.`